sdk/src/tx/retryTxSender.ts

```diff
@@ -56,7 +56,7 @@ export class RetryTxSender implements TxSender {
 			opts = this.provider.opts;
 		}
 
-		const signedTx = preSigned
+		const signedTx = preSigned && tx.feePayer
```

Comment:
if it's preSigned, why would it not have a feePayer? seems like there is an issue upstream?

Reply:
no issue, just sort of a redundancy check but i can remove